README.md

```diff
@@ -71,6 +71,9 @@ Using the `traveler` object, update the DOM to show the traveler's *name*, *nick
 
 **YOUR NOTES**
 ```
+Can acces data.js because it's in same file directory, or maybe because of
+<script src="data.js"></script> in index.html? Not sure
```

Comment:
It's because of the <script> tag! Any Javascript files that we want loaded in the browser must be added with a <script> tag.